docs/src/reference.md

```diff
@@ -0,0 +1,100 @@
+# Algorithms
+
+This section contains all algorithms of the [`FrankWolfe.jl`](https://github.com/ZIB-IOL/FrankWolfe.jl) package.
+
+## Functions
+
+```@docs
+frank_wolfe
+lazified_conditional_gradient
+away_frank_wolfe
+blended_conditional_gradient
+```
+
+
+# LMOs
+
+The Linear Minimization Oracle (LMO) is an integral part of the iterative step in the FW algorithm. Given $d\in \mathcal{X}$, it returns:
```

Comment:
does the $d \in$ display correctly?

Reply:
Yes locally it does (I did that before I knew about the `` notation). I'll just switch it to double ticks to be more consistent though.